cmds/tui.c

```diff
@@ -2368,12 +2658,33 @@ static void display_graph_field(WINDOW *win, int selected_field, bool graph_fiel
 	for (i = 0; i < NUM_GRAPH_FIELD; i++) {
 		if (i == selected_field)
 			wattron(win, A_REVERSE);
-		mvwprintw(win, i + ARRAY_SIZE(graph_field_help) + 4, 2, "[ %c ] %s",
+		mvwprintw(win, i + ARRAY_SIZE(field_help) + 4, 2, "[ %c ] %s",
 				graph_field_flags[i] ? 'x' : ' ', graph_field_names[i]);
 		wattroff(win, A_REVERSE);
 	}
 }
 
+static void display_report_field(WINDOW *win, int selected_field, bool report_field_flags[])
+{
+	int i;
+
+	for (i = 0; i < NUM_REPORT_FIELD; i++) {
+		if (i == selected_field)
+			wattron(win, A_REVERSE);
+		mvwprintw(win, i + ARRAY_SIZE(field_help) + 4, 2, "[ %c ] %s",
+				report_field_flags[i] ? 'x' : ' ', report_field_names[i]);
+		wattroff(win, A_REVERSE);
+	}
+}
+
+static inline void display_graph_or_report_field(WINDOW *win, int selected_field, bool graph_field_flags[], bool report_field_flags[], enum mode mode)
```

Comment:
Both the line length and function are too long IMO.  Maybe it can be display_tui_field() and break the line around 80 column.